tests/onnx/test_onnx_v2.py

```diff
@@ -181,73 +181,70 @@ def test_values_override(self):
 
 PYTORCH_EXPORT_MODELS = {
     ("albert", "hf-internal-testing/tiny-albert"),
-    ("bert", "bert-base-cased"),
-    ("big-bird", "google/bigbird-roberta-base"),
-    ("ibert", "kssteven/ibert-roberta-base"),
-    ("camembert", "camembert-base"),
-    ("clip", "openai/clip-vit-base-patch32"),
-    ("convbert", "YituTech/conv-bert-base"),
-    ("codegen", "Salesforce/codegen-350M-multi"),
-    ("deberta", "microsoft/deberta-base"),
-    ("deberta-v2", "microsoft/deberta-v2-xlarge"),
-    ("convnext", "facebook/convnext-tiny-224"),
-    ("detr", "facebook/detr-resnet-50"),
-    ("distilbert", "distilbert-base-cased"),
-    ("electra", "google/electra-base-generator"),
-    ("resnet", "microsoft/resnet-50"),
-    ("roberta", "roberta-base"),
-    ("roformer", "junnyu/roformer_chinese_base"),
-    ("squeezebert", "squeezebert/squeezebert-uncased"),
-    ("mobilebert", "google/mobilebert-uncased"),
-    ("mobilevit", "apple/mobilevit-small"),
-    ("xlm", "xlm-clm-ende-1024"),
-    ("xlm-roberta", "xlm-roberta-base"),
-    ("layoutlm", "microsoft/layoutlm-base-uncased"),
-    ("layoutlmv3", "microsoft/layoutlmv3-base"),
-    ("groupvit", "nvidia/groupvit-gcc-yfcc"),
-    ("levit", "facebook/levit-128S"),
-    ("owlvit", "google/owlvit-base-patch32"),
-    ("vit", "google/vit-base-patch16-224"),
-    ("deit", "facebook/deit-small-patch16-224"),
-    ("beit", "microsoft/beit-base-patch16-224"),
-    ("data2vec-text", "facebook/data2vec-text-base"),
-    ("data2vec-vision", "facebook/data2vec-vision-base"),
-    ("perceiver", "deepmind/language-perceiver", ("masked-lm", "sequence-classification")),
-    ("perceiver", "deepmind/vision-perceiver-conv", ("image-classification",)),
-    ("longformer", "allenai/longformer-base-4096"),
-    ("yolos", "hustvl/yolos-tiny"),
-    ("segformer", "nvidia/segformer-b0-finetuned-ade-512-512"),
+    ("beit", "hf-internal-testing/tiny-random-beit"),
```

Comment:
I decided to order these alphabetically because it was annoying to find the model names quickly :)